src/lib/dnssd/Discovery_ImplPlatform.cpp

```diff
@@ -36,10 +36,94 @@
 namespace chip {
 namespace Dnssd {
 
-DiscoveryImplPlatform DiscoveryImplPlatform::sManager;
+namespace {
```

Comment:
This mostly looks like the classes got moved from being private members to being just in the anonymous namespace. What was the reason for this change? And what parts of the code actually changed here? I'm trying to mentally diff it, but I want to make sure I didn't miss anything - looks like it's just calling back to the device proxy rather than the DiscoveryImplPlatform class. Is that the only change?

Reply:
The main motivation was that I didn't want to add an intermediate class between `ResolverProxy` and those methods.
And since those methods are only for the platform use case they will just be empty for the minimal use case.